docs/install-script.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This page is written in Japanese.

```suggestion
<html lang="ja">
```